how-to/qemu-riscv.rst

```diff
@@ -35,53 +45,56 @@ Using the pre-installed server image
        :matches: (riscv64\.img)
 
    .. on jammy, use the +unmatched image for QEMU; later releases should use
-      the unsuffixed images, hence the horrid regex above
+      the unsuffixed images, hence the horrid regex above.
 
 #. Unpack the image:
 
    .. code-block:: text
 
-       xz -dk ubuntu-24.04.2-preinstalled-server-riscv64.img.xz
+       xz -dk ubuntu-preinstalled-server-riscv64.img.xz
 
 
 #. Optionally, if you want a larger disk, you can expand the disk (the
    file-system will be automatically resized too):
 
    .. code-block:: text
 
-       qemu-img resize -f raw ubuntu-24.04-preinstalled-server-riscv64.img +5G
+       qemu-img resize -f raw ubuntu-preinstalled-server-riscv64.img +5G
 
 
-#. Next use u-boot-qemu to boot the virtual machine. A working example with all
+#. Next use U-Boot to boot the virtual machine. A working example with all
```

Comment:
u-boot-qemu is a package name.

Reply:
Hmm, I can't get the point to use a package name here. We have already put the "package - thing" list on the top of this article, so readers should know what "U-Boot" is.

Could you please point me out?